src/sctp.rs

```diff
@@ -585,6 +602,8 @@ const CHUNK_TYPE_I_FORWARD_TSN: u8 = 0xc2;
 
 const INIT_ACK_PARAM_STATE_COOKIE: u16 = 0x07;
 const INIT_PARAM_FORWARD_TSN: u16 = 0xc000;
+const INIT_PARAM_SUPPORTED_EXTENSIONS: u16 = 0x8008;
+const INIT_PARAM_EXT_FORWARD_TSN: u8 = 0xc0;
```

Comment:
I couldn't find a confirmation that it's valid to communicate the forward-tsn chunk support like this. Though I was able to find that `I-FORWARD-TSN` is valid to be communicated like this (according to https://datatracker.ietf.org/doc/html/rfc8260#section-5.2).